src/blocks/_fhi-material-tree.block.scss

```diff
@@ -68,6 +79,30 @@
         display: none;
     }
 
+    &--check {
```

Comment:
As mentioned [here](https://github.com/folkehelseinstituttet/Fhi.Frontend.Demo/blob/98932da12d435d83439aebdcd7fe7e7a66b2e19e/doc/scss-example.md?plain=1#L18)  ; I'm a fan of making global modifiers for a block outside of the block...